client/src/lib/types.ts

```diff
@@ -0,0 +1,20 @@
+export type Page = {
```

Comment:
I personally have a very strong preference for putting the types in the file with the code that naturally "owns" the type. I dislike a generic `types.ts` file. It makes me feel like I haven't properly organized my code.

You are just starting the project, though, and there's not currently a natural place to put these types. Hopefully as you flesh out more components it will become more clear where these should go.